pkgs/development/python-modules/diofant/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchPypi
+, glibcLocales, pytestrunner, setuptools_scm
+, mpmath, isort, strategies
+}:
+
+buildPythonPackage rec {
+  pname = "diofant";
+  version = "0.10.0";
+
```

Comment:
```suggestion
    disabled = pythonOlder "3.5";
    
```

Reply:
Isn't `isPy3k` ok?

Reply:
Sorry, I commented on an earlier version of your commit that didn't have the `isPy3k` in it.
Yes, `isPy3k` is OK, since we don't have 3 < python < 3.5.